src/main/java/templates/groups.soy

```diff
@@ -14,23 +14,45 @@
         </head>
 
         <body>
-            <div>
-                <h1 class="heading mdc-typography--headline3">Movements</h1>
-            </div>
-                
-            <div class="card-gallery">
-                
-                // For each group, a new card is rendered with that group's info
-                {for $groupMap in $groups}
-                    {call .groupCard}
-                        {param image: $groupMap['groupImage'] /}
-                        {param name: $groupMap['groupName'] /}
-                        {param description: $groupMap['groupDescription'] /}
+            <aside class="mdc-drawer" id="nav-drawer" >
+                <div class="mdc-drawer__content">
+                    <nav class="mdc-list" style="padding: 0">
+                        <div class="mdc-list-item mdc-list-item--activated" style="margin-top: 10px" aria-current="page">
```

Comment:
Was the styling part included already or did you add that? If not try to put that in css file